requirements.txt

```diff
@@ -1,3 +1,4 @@
+itsdangerous==2.0.1
```

Comment:
What is this 😧 ?

Reply:
es algo que queria comentar. Un paquete que instala flask. Pip los tests reclamaron por la version del paquete (en algun punto se depreco _json de itsdangerous).

Reply:
me costo pero enccontre el [error en los tests](https://github.com/alercebroker/ztf_api/runs/5451866112?check_suite_focus=true)